README.md

```diff
@@ -535,3 +535,83 @@ func main() {
 	fmt.Printf("Transaction ID: %s\n", sendResponse.TxID)
 }
 ```
+
+## Manipulating multisig transactions
+
+Here, we first create a simple multisig payment transaction,
+with three public identities and a threshold of 2:
+
+```golang
+addr1, _ := types.DecodeAddress("DN7MBMCL5JQ3PFUQS7TMX5AH4EEKOBJVDUF4TCV6WERATKFLQF4MQUPZTA")
+addr2, _ := types.DecodeAddress("BFRTECKTOOE7A5LHCF3TTEOH2A7BW46IYT2SX5VP6ANKEXHZYJY77SJTVM")
+addr3, _ := types.DecodeAddress("47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU")
+ma, err := MultisigAccountWithParams(1, 2, []types.Address{
+	addr1,
+	addr2,
+	addr3,
+})
+if err != nil {
+	panic("invalid multisig parameters")
+}
+fromAddr, _ := ma.Address()
+txn, err := MakePaymentTxn(
+	fromAddr.String(),
+	"INSERTTOADDRESHERE",
+	10,     // fee per byte
+	10000,  // amount
+	100000, // first valid round
+	101000, // last valid round
+	nil,    // note
+	"",     // closeRemainderTo
+	"",     // genesisID
+)
+txid, txBytes, err := SignMultisigTransaction(secretKey, ma, txn)
+if err != nil {
+	panic("could not sign multisig transaction")
+}
+fmt.Printf("Made partially-signed multisig transaction with TxID %s: %x\n", txid, txBytes)
+
+```
+
+Now, we can write the returned bytes to disk:
+```golang
+file, _ := os.Create("./arbitrary_file.tx")
```

Comment:
nothing incorrect here, but I'm a bigger fan of `ioutil.WriteFile` which would do these three lines in one command.